### PR TITLE
fix trainer import error on ce

### LINKED ISSUE #10451: 新加目录paddle.fluid.transpiler没有修改setup.py.in

### PR DESCRIPTION
**Pull request by Yancey0623**
Fixed #10451

### CODE REVIEW

**Review by Superjomn — DISMISSED** [review]
LGTM

**Review by dzhwinter — APPROVED** [review]
LGTM

File: python/paddle/fluid/__init__.py [hunk 1]
```diff
@@ -60,6 +60,7 @@
     'io',
     'initializer',
     'layers',
+    'transpiler'
```
Comment (reyoung) [inline python/paddle/fluid/__init__.py:63]: It seems that a comma is missing here...

Comment (Yancey0623) [reply python/paddle/fluid/__init__.py:63]: Sorry for the mistake... will fix it.

**Pull request merged**